src/tutorials/CliSK/5 - Best practices/5.1 - Coding standards.md

```diff
@@ -0,0 +1,26 @@
+A yarn script is provided to lint the code. It will respect most of our coding standards. Simply use the `yarn lint --fix` command
+#### Logs
+
+Regarding the logs you might put in your code, it is recommended to avoid logging sensible information once the konnectors is ready to go to production.
```

Comment:
```suggestion
Regarding the logs you might put in your code, it is recommended to avoid logging sensible information once the konnector is ready to go to production.
```